source/ShipJumpNavigation.cpp

```diff
@@ -225,7 +218,7 @@ void ShipJumpNavigation::ParseOutfit(const Outfit &outfit)
 		double distance = outfit.Get("jump range");
 		if(distance <= 0.)
 			distance = System::DEFAULT_NEIGHBOR_DISTANCE;
-		double cost = CalculateFuelCost(DEFAULT_JUMP_DRIVE_COST);
+		double cost = CalculateFuelCost();
 
 		UpdateJumpDriveCosts(distance, cost);
 	}
```

Comment:
This is going to behave differently if an outfit is both an HD and JD. Previously, it would have different costs for the two jumps, but now they both use the JD's default cost.

Reply:
I think that's fine. "hyperdrive fuel" and "jump drive fuel" both defaulting to the "jump fuel" value. I don't think we need a separate value for the scram drive.